app/forms/invite_form.rb

```diff
@@ -0,0 +1,6 @@
+class InviteForm < Reform::Form
+  property :email
+  validates :email, :presence => true
+  validates :email, :email => true
+  validates_uniqueness_of :email
```

Comment:
I can see a use case where someone needs more than one invite. Maybe leave this out for now?